src/SFA.DAS.Reservations.Web/SFA.DAS.Reservations.Web.csproj

```diff
@@ -40,6 +41,7 @@
     <PackageReference Include="SFA.DAS.Http" Version="3.2.28" />
     <PackageReference Include="SFA.DAS.NLog.Targets.Redis" Version="1.1.5" />
     <PackageReference Include="SFA.DAS.Provider.Idams.Stub.Extensions" Version="1.0.14" />
+    <PackageReference Include="StackExchange.Redis" Version="2.2.4" />
```

Comment:
Just one thing to note, on das-recruit, I had to downgrade the StackExchange.Redis due to other dependency mismatch occurring at runtime. If the application runs fine locally then there shouldn't be concern, but if there is an issue, could try downgrading like [here]( https://github.com/SkillsFundingAgency/das-recruit/blob/360316ebb7da2ccc3b15a4a136b38fc1718fbc84/src/Shared/Recruit.Shared.Web/Recruit.Shared.Web.csproj#L12):